CRM/Civiquickbooks/Invoice.php

```diff
@@ -933,61 +935,38 @@ protected function generateTaxDetails($line_items) {
    * @throws \CiviCRM_API3_Exception
    */
   protected function findPushContributions($params, $limit) {
-    $criteria = [
-      'accounts_needs_update' => 1,
-      'plugin' => $this->plugin,
-      'connector_id' => 0,
-      'accounts_status_id' => ['NOT IN', 3],
```

Comment:
@agileware-fj It was this line that was breaking things I think because 3 is not an array